content/blog/2022-10-06-interview-white-rabbit/index.en.md

```diff
@@ -0,0 +1,326 @@
+---
+title: 'Interview: White rabbit'
+author: 
+  - Open make interview team
+  - Javier Serrano
+  - Amanda Diez Fernandez
+date: '2022-05-01'
+slug: interview-white-rabbit
+categories: [interview]
+tags:
+  - technology
+  - open hardware
+  - personal story
+---
+Interview: White rabbit, by the Open make team, Javier Serrano and Amanda Diez Frenandez.
+copyright to the authors, distributed under a CC-BY 4.0 licence.
+
+Sections:
+- [The project](#the-project)
+- [The Hardware](#the-hardware)
+- [The Research outputs](#research-outputs)
+- [The participants](#participants)
+
+
+>Interviewee: Javier Serrano (CERN) & Amanda Diez Fernandez (CERN)
+>
+>Interviewers: Robert Mies (TU Berlin) & Moritz Maxeiner (FU Berlin)
+Transcription and editing: Diana Paola Americano Guerrero, Robert Mies, Moritz Maxeiner & Julien Colomb
+
+<img src="images/Screenshot 2022-10-06 at 14.16.40.png" alt="screenshot of the interview" width="60%"/>
+
+*Screenshot of the interview.*
+
+At CERN, the European Laboratory for Particle Physics, researchers perform experiments in which particle beams are passed through a series of increasingly higher energy accelerators, until they reach the LHC, the Large Hadron Collider, which is currently the biggest accelerator at CERN. There, the beams are made to collide so the physicists can study the outcome.
+
+From an engineering perspective, this requires a distributed real-time system to ensure all components are synchronized at sub-nanosecond accuracy - something previously existing solutions were unable to provide. The project to develop the necessary hard- and software as open source, spearheaded at CERN by a team of electronics designers and Linux kernel developers around Javier Serrano (section leader), became known as “White Rabbit”. Amanda Diez has joined the project recently to increase internal and external communitation efforts.
+
+{{< card  "The White Rabbit in a nutshell">}}
+
+![image alt](https://ohwr.org/project/wr-switch-hw/uploads/8bec64c2bc708e20d516d4ddb9ce8da4/wrs3_W02-v3.4-smc.jpg "WR Switch v3.4: Front view, FMC connectors") 
+*WR Switch v3.4: Front view, FMC connectors, shared under [no license?]*
```

Comment:
Circuit board and relevant mechanics are licensed under `CERN OHL v.1.2`.

See https://ohwr.org/project/wr-switch-hw/tree/master/circuit_board/wrs_v3 and https://ohwr.org/project/wr-switch-hw/blob/master/mechanics/v3.4/pdf/01%20-%20Front%20and%20top%20view.pdf

Reply:
If you mean under which license the image is shared under, I would write `, by Grzegorz Daniluk and Erik van der Bij`, as they are the ones listed at the bottom of the documentation page where the images are: https://ohwr.org/project/wr-switch-hw/wikis/home